helm-service/chart/values.yaml

```diff
@@ -5,6 +5,8 @@ helmservice:
     tag: ""                                  # Container Tag
   service:
     enabled: true                            # Creates a Kubernetes Service for the helm-service
+  gracePeriod: 90                            # PreStop hook +30s
+  preStop: 60
```

Comment:
i would suggest a slightly more descriptive name like `preStopHookTime` or so?
Something that clearly shows that this is a time value